apt/private/resolve.bzl

```diff
@@ -17,15 +17,33 @@ echo ''
 echo 'Writing lockfile to {workspace_relative_path}' 
 cp $lock {workspace_relative_path}
 
+# Detect which file we wish the user to edit
+if [ -e $BUILD_WORKSPACE_DIRECTORY/WORKSPACE ]; then
```

Comment:
looks like we should also support MODULE.bazel here.

Reply:
yes, will file a follow-up